analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectFunctionalTest.java

```diff
@@ -41,9 +49,8 @@
 
 @RunWith(BMUnitRunner.class)
 public class SimpleProjectFunctionalTest extends AbstractWiremockTest {
-
     @Rule
-    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog().muteForSuccessfulTests();
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog()/*.muteForSuccessfulTests()*/;
```

Comment:
TODO: revert

Reply:
Shall we change this now?

Reply:
I planned to once all outstanding issues were fixed. This helps to see issues related to the reports right on the console.

Reply:
Ok